REST_API_Development/PictogramFiles.md

```diff
@@ -0,0 +1,35 @@
+# Pictogram files
+
+As of May of 2019, the `web-api` has no pictograms stored in the repository. Some pictograms have been stored in the [DBInitializer](https://github.com/aau-giraf/web-api/blob/develop/GirafRest/Data/DBInitializer.cs) class but have since been removed, because storing images in a database is generally considered harmful. 
```

Comment:
```suggestion
As of May 2019, the [web-api](https://github.com/aau-giraf/web-api) has no pictograms stored in the repository. Some pictograms have been stored in the [DBInitializer](https://github.com/aau-giraf/web-api/blob/develop/GirafRest/Data/DBInitializer.cs) class but have since been removed, because storing images in a database is generally considered harmful. 
```